infra/gcp/boskos/main.tf

```diff
@@ -1,12 +1,12 @@
 
 
-terraform {
-  required_providers {
-    google = {
-      version = "4.15"
-    }
-    google-beta = {
-      version = "4.15"
-    }
-  }
-}
+# terraform {
```

Comment:
Lots of commenting out through out this PR, is that necessary? Can't we just remove and if we need it back we can revert?

Reply:
Fixed

Reply:
Oops forgot to push, afk

Reply:
This is ready now.